test/core/node.js

```diff
@@ -0,0 +1,5 @@
+'use strict'
+
+require('./pin')
+require('./pin-set')
+// require('./key-exchange')
```

Comment:
Does anyone know why these tests were never being run? @richardschneider ??

Reply:
@diasdavid No idea, the tests should be run.

Reply:
They weren't being picked up anywhere. I'll add them as a separate PR then.